samples/RenderDemo/Pages/PathMeasurementPage.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Diagnostics;
+using System.Drawing.Drawing2D;
+using System.Security.Cryptography;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.LogicalTree;
+using Avalonia.Media;
+using Avalonia.Media.Imaging;
+using Avalonia.Media.Immutable;
+using Avalonia.Threading;
+using Avalonia.Visuals.Media.Imaging;
+
+namespace RenderDemo.Pages
+{
+    public class PathMeasurementPage : Control
+    {
+        private RenderTargetBitmap _bitmap;
+
+        protected override void OnAttachedToLogicalTree(LogicalTreeAttachmentEventArgs e)
+        {
+            _bitmap = new RenderTargetBitmap(new PixelSize(500, 500), new Vector(96, 96));
+            base.OnAttachedToLogicalTree(e);
+            AffectsRender<PathMeasurementPage>(BoundsProperty);
```

Comment:
To avoid giving people wrong ideas from samples - this `AffectsRender` should be in the static ctor instead.